### PR TITLE
[doc] reorganize file structure and add file title (For RC1 release)

### PR DESCRIPTION
**Pull request by whitewum**

### CODE REVIEW

**Comment by whitewum** [comment]
close #883

File: docs/manual-EN/language-structure/literal-values/numeric-literals.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
-Number literals include interger literals and floating-point literals.
+# Numeric Literals
+
+Number literals include interges literals and floating-point literals (doubles).
+
+## Interges literals
```
Comment (Amber1990Zhang) [inline docs/manual-EN/language-structure/literal-values/numeric-literals.md]: Ditto.
Integers Literals
Please make the titles consistent in form.

File: docs/manual-EN/language-structure/literal-values/numeric-literals.md [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 
 Integers are 64 bit digitals, and can be preceded by + or - to indicate a positive or negative value, respectively. They're the same as `int64_t` in the C language.
 
-floating-points are the same as `double` in the C language.
+Notice that the maximum value for the positive interges is `9223372036854775807`. It's syntax-error if you try to input any value larger than the maximum. So as the minimum value `-9223372036854775808` for the negative interges.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/language-structure/literal-values/numeric-literals.md]: `integer`, not `interge`

File: docs/manual-EN/language-structure/literal-values/numeric-literals.md [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+
+## Floating-point literals (doubles)
```
Comment (Amber1990Zhang) [inline docs/manual-EN/language-structure/literal-values/numeric-literals.md]: Ditto, title form.

File: docs/manual-EN/language-structure/literal-values/numeric-literals.md [hunk 4]
```diff
@@ -1,18 +1,26 @@
...
+
+Floating-points are the same as `double` in the `C` language.
+
+The range for double is about `-1.79769e+308` to `1.79769e+308`.
+
+Scientific notations is not supported yet.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/language-structure/literal-values/numeric-literals.md]: Scientific notation

File: docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md [hunk 5]
```diff
@@ -1,28 +1,31 @@
...
+    (<create_definition>, ...)
     [tag_edge_options]
   
-create_definition:
-    prop_name data_type
-    
-tag_edge_options:
-    option [, option ...]
+<create_definition> ::=
+    <prop_name> <data_type>
```
Comment (Amber1990Zhang) [inline docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md]: Should we add `muitlple properties should be separated by comma`?

File: docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md [hunk 6]
```diff
@@ -1,28 +1,31 @@
...
-option:
-    TTL_DURATION [=] ttl_duration
-    | TTL_COL [=] prop_name
+<option> ::=
+    TTL_DURATION [=] <ttl_duration>
+    | TTL_COL [=] <prop_name>
+    | DEFAULT <default_value>
 ```
 
 Nebula's graph schema is composed of tags and edges, either of which may have properties. `CREATE TAG` statement defines a tag with the given name. `CREATE EDGE` statement is to define an edge type.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md]: `CREATE TAG` statement defines a tag with the given name while `CREATE EDGE` defines an edge type.

File: docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md [hunk 7]
```diff
@@ -1,28 +1,31 @@
...
 
 There are several aspects to this syntax, described under the following topics in this section:
 
+<!---
 * [Tag Name and Edge Type Name](#tag-name-and-edgetype-name)
 
 * [Property Name and Data Type](#property-name-and-data-type)
+-->
 
 ### Tag Name and EdgeType Name
```
Comment (Amber1990Zhang) [inline docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md]: Suggestion: Edge Type should be two words

File: docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md [hunk 8]
```diff
@@ -42,6 +45,12 @@ There are several aspects to this syntax, described under the following topics i
     
     > NULL and NOT NULL constrain are not supported yet when creating tags/edges (comparing with relational databases).
 
+* **default values**
+
+    You can set the default value of a property when creating a tag/edge. When inserting a new vertex or edge, you don't have to provide the value for that property, or you can write a user-specify value if you don't want to use the default one.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md]: please delete `or`

**Comment by dutor** [comment]
For testing on _skip building_

Jenkins, go

**Comment by dutor** [comment]
Jenkins, go

File: docs/manual-EN/functions-and-operators/logical-operators.md [hunk 9]
```diff
@@ -1,8 +1,11 @@
+# Logical Operators
+
 |  ***Name***    |  ***Description***    | 
 |:----|:----:|
 |   &&     |   Logical AND     | 
-|   !    |   Logical NOT     | 
+|   !      |   Logical NOT     | 
 |   \|\|   |   Logical OR   | 
+|   ^      |   Logical XOR  |
```
Comment (laura-ding) [inline docs/manual-EN/functions-and-operators/logical-operators.md]: `XOR` representing a logical operation, `^` representing an arithmetic operation

**Pull request merged**